### ISSUE TITLE
Subsection 6.3: `simple_sync_workflow.py` confusion

### ISSUE THREAD

**Issue by yakutovicha** [aiidateam/aiida-tutorials#70]
Subsection 6.3 speaks about the file `simple_sync_workflow.py`, but in fact there are two files of this name: one that is proposed for downloading (I) and another that is located in `./examples/aiida-demos/tutorial/scripts/` (II).

There are couple of problems related to them:
* They are different (at least in the version of workhorse I am using)
* `result = run(PwCalculation, **inputs)` is present in (II).
* There is no `tutorial_scripts` directory:
> Run the workflow by running the following command from the `tutorial_scripts` directory
* codename in (II) is still wrong (see also #69 )

**Comment by sphuber** [comment]
All references to `tutorial_scripts` and `aiida-demos` should be removed. This was the old setup and all the example files are now contained in the `aiida-tutorials` repo and made available through download buttons. This ensure that they no longer have to be uploaded to the VM and makes sure that people following the tutorial without the VM also have access to them. The script that is currently in `aiida-tutorials` should be leading

**Comment by yakutovicha** [comment]
The line
```
result = run(PwCalculation, **inputs)
```
is still in the text. But in the script file it is:
```
calculations[label] = run(PwCalculation, **inputs)
```

**Comment by sphuber** [comment]
It should be the latter. Sorry, I missed that one

**Comment by sphuber** [comment]
Fixed in PR #84
